modules/ztp-single-node-clusters-factory.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-single-node-clusters_{context}"]
+
+= Single node clusters
+
+You use zero touch provisioning (ZTP) to deploy single node clusters at disconnected far edge sites. A single node cluster runs {product-title} on top of one bare metal machine, hence the single node. Edge servers contain a single node with supervisor functions and worker functions on the same host that are deployed at low bandwidth or disconnected edge sites.
+
+{product-title} is configured on the single node to use workload partitioning. Workload partitioning separates cluster management workloads from
+user workloads and can run the cluster management workloads on a reserved set of CPUs.
+Workload partitioning is useful for resource-constrained environments, such as single-node production deployments,
```

Comment:
You hyphenate single-node here but elsewhere it isn't hyphenated. Should it be hyphenated in general?